packages/botonic-plugin-contentful/tests/contentful/manage/manage-contentful-entry.test.ts

```diff
@@ -75,4 +80,44 @@ describe('ManageContentful entries', () => {
       sut.deleteContent(context, TEST_NEW_CONTENT_ID)
     ).rejects.toThrow(CmsException)
   })
+
+  test('TEST: convertValueType converts the ButtonStyle and the FollowUp', async () => {
+    const sut = testManageContentful()
+    const contentful = testContentful({ disableCache: true })
+    const NEW_CONTENT = new TopContentId(ContentType.TEXT, rndStr())
+    const FOLLOW_UP_CONTENT = new TopContentId(ContentType.TEXT, rndStr())
+    const name = rndStr()
+    const text = rndStr()
+    const followUpName = rndStr()
+    try {
+      // ACT
+      await sut.createContent(context, NEW_CONTENT.model, NEW_CONTENT.id)
+      await sut.createContent(
+        context,
+        FOLLOW_UP_CONTENT.model,
+        FOLLOW_UP_CONTENT.id
+      )
+      await sut.updateFields(context, NEW_CONTENT, {
```

Comment:
I'm afraid that this might fail because the the newly created contents have not yet been able to publish to the CDN.
Try to execute the test at least 10 times (you can do it from terminal with a bash script copy pasting the ts-jest command), and if it fails, wrap the 2 updateFields calls within a repeatWithBackoff. If it fails in tests, it will might also fail from flowbuilder. In this case,  I suggest creating a PR in which createContent has an optional argument to pass the fields so that a single call to contentful API is enough and then you can't have a timeout. 

Another option would be the following:
 create a function at manage-contentful.helper which, within repeatWithBackoff, executes manageCms and later checks that the 
```
asyn cfunction createContent(
    manageCms: ManageCms,
    context: ManageContext,
    model: ContentType,
    id: string
  ): Promise<void> 
```

Reply:
I've executed it 20 times and it didn't fail